scripts/sync_i18n.py

```diff
@@ -4,52 +4,48 @@
 # Script to synchronize each language pack's items against Wowchemy's master pack (English).
 # https://wowchemy.com
 #
-# Prerequisites: pip3 install PyYAML
+# Prerequisites: pip3 install ruamel.yaml
 #
-# TODO: Switch from PyYAML to Ruamel in order to load/dump comments -
-#  see https://stackoverflow.com/questions/47382227/python-yaml-update-preserving-order-and-comments
+# CHANGES:
```

Comment:
You can remove all of this TODO/CHANGES section and replace it with a note for future contributors.

For example,

> Note: we use Ruamel rather than PyYAML in order to preserve comments in language packs